File: common/src/main/java/net/vanillaoutsider/betterdogs/client/BetterDogsClient.java
package net.vanillaoutsider.betterdogs.client;

import net.fabricmc.api.ClientModInitializer;
import net.vanillaoutsider.betterdogs.BetterDogs;
import org.slf4j.LoggerFactory;

public class BetterDogsClient implements ClientModInitializer {

    @Override
    public void onInitializeClient() {
        BetterDogs.LOGGER.info("Better Dogs client initialized");
        // Client-side initialization (particles are handled in mixin)
    }
}

File: common/src/client/kotlin/net/vanillaoutsider/betterdogs/BetterDogsClient.kt
package net.vanillaoutsider.betterdogs

import net.fabricmc.api.ClientModInitializer
import org.slf4j.LoggerFactory

object BetterDogsClient : ClientModInitializer {
    private val logger = LoggerFactory.getLogger(BetterDogs.MOD_ID)
    
    override fun onInitializeClient() {
        logger.info("Better Dogs client initialized")
        // Client-side initialization (particles are handled in mixin)
    }
}
